mentors/src/data_pipeline/pipeline_entrypoint.py

```diff
@@ -112,7 +112,7 @@ def download_mentor_data(url, mentor):
 
         while part_found:
             part += 1
-            print(f"INFO: Downloading session {session} part {part}")
+            print(f"INFO: Downloading session {session} part {part} data")
```

Comment:
could we make this `INFO: Downloading data session {session} part {part} (if exists)` ?